### PR TITLE
Add a spinner to Croud

### PR DESCRIPTION
**Pull request by SStorm**
## Summary of the changes / Why this is an improvement

This will show up in stderr and clear just before any output is printed. Just a nice gimmick.

https://user-images.githubusercontent.com/4594061/107764893-d71c2f00-6d28-11eb-9ad2-21417bce7545.mp4

## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Comment by lukasbals** [comment]
I like! 🚀

File: CHANGES.rst [hunk 1]
```diff
@@ -5,6 +5,8 @@ Changes for croud
 Unreleased
 ==========
 
+- Added a spinner for some nicer UX.
```
Comment (lukasbals) [inline CHANGES.rst:8]:
```suggestion
- Added a loading indicator for some nicer UX.
```

**Review by lukasbals — APPROVED** [review]
Nice work!

**Comment by turbo-ele** [comment]
😎

**Review by turbo-ele — APPROVED** [review]
😎

**Review by WalBeh — COMMENTED** [review]
❤️

**Pull request merged**